TACC/Core-Portal
# task/TV3-102: Implement relative homeDir listings for Data Files systems

## Overview

- Refactor data files configuration to a simplified, unified list in `_PORTAL_DATAFILES_STORAGE_SYSTEMS`
- Remove unused managers and onboarding steps made redundant in Tapis v3

## Related

* [TV3-102](https://jira.tacc.utexas.edu/browse/TV3-102)

## Testing

1. Click around data files systems and confirm all buttons and navigations works
2. Submit a job from the Hello World application and confirm notifications come through, the job archives successfully, and you can browse the archive directory from the archive link in the job detail view

## UI

![Screen Shot 2023-01-12 at 6 05 05 PM](https://user-images.githubusercontent.com/20326896/212206818-1e523a67-b792-45d9-969d-1e37883ecff1.png)
